app/services/idv/inherited_proofing/mock/va_api_mock_client.rb

```diff
@@ -0,0 +1,50 @@
+module Idv::InheritedProofing
+  module Mock
+    class VaApiMockClient
```

Comment:
No big deal, only food for thought :) but you might want to think about using the namespace/class/file name: `Idv::InheritedProofing::Mocks::Va::Api` `user_attributes.rb` or `service.rb` (for example). "Mock" anywhere in the file name is redundant as it's already part of your namespace.

If we ever needed supporting classes/mixins/etc. for any of our API mocks, we have a place to put them that make sense; as it stands, we'd be forced to stuff everything in `app/services/idv/inherited_proofing/mock/` regardless of service provider.

Reply:
@gangelo Looking at this https://github.com/18F/identity-idp/blob/main/app/services/doc_auth/mock/doc_auth_mock_client.rb, I believe this is okay ?

Reply:
Hi @rnagilla-gsa the difference with inherited proofing (IP), is that IP will have (at some point) multiple service provides that will need mocks. The example pattern you shared has supporting classes in the `mocks` folder. For IP, we will be forced to stuff all the (tentative) future mocks and supporting codez for our service provides in this folder, or resort to service provider subfolders (which is what I am proposing to begin with). Not something to lose sleep over now, but at some point we may be forced to restructure and refactor as we gain more service providers and need to create more mocks. Just wondering if we should establish the structure now, rather than later, that's all. No biggie either way.